pkgs/by-name/mi/mixtool/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "mixtool";
+  version = "unstable-2025-07-07";
```

```suggestion
  version = "0-unstable-2025-07-07";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning